ledger-core/virtual/integration/benchs/basic_test.go

```diff
@@ -82,8 +83,107 @@ func BenchmarkVCallRequestGetMethod(b *testing.B) {
 	}
 }
 
-func BenchmarkTestAPIGetBalance(b *testing.B) {
+func BenchmarkVCallRequestAcceptMethod(b *testing.B) {
+	convlog.DisableTextConvLog()
+	server, ctx := utils.NewServer(nil, b)
+	defer server.Stop()
+	server.IncrementPulseAndWaitIdle(ctx)
+
+	var (
+		class  = walletproxy.GetClass()
+		object = reference.NewSelf(server.RandomLocalWithPulse())
+	)
 
+	walletMemory := insolar.MustSerialize(testwallet.Wallet{
+		Balance: 1234567,
+	})
+
+	content := &payload.VStateReport_ProvidedContentBody{
+		LatestDirtyState: &payload.ObjectState{
+			Reference: reference.Local{},
+			Class:     class,
+			State:     walletMemory,
+		},
+	}
+
+	report := &payload.VStateReport{
```

Comment:
Could u please provide 'AsOf' filed, it will strict required. Also, increment pulse before sending state report, we should get report from the previous pulse not in current